boards/stm32l0538-disco/include/periph_conf.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_stm32l0538-disco
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the STM32L0538-DISCO board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_rtt_default.h"
```

Comment:
This is not valid, it is calculated based on the CLK beeing 32Khz, but here it is 37Khz. I created a branch to patch this https://github.com/fjmolinas/RIOT/tree/pr_stm32_rtt_lsi.

The problem is the test still fails because LSI isn't very precise it can have a pretty big deviation around 37KhZ.

Reply:
You are right and your branch is useful and should be PRed I think.